package.json

```diff
@@ -69,6 +69,7 @@
     "lottie-react": "^2.3.1",
     "netlify-cli": "^7.1.0",
     "nodemon": "^2.0.12",
+    "playwright": "1.37.1",
```

Comment:
If we're adding this as a dependency, it may be worth revisiting how we trigger other integration tests runs (see the `playwright:{foo}` scripts below). (You may also want to add a script to specifically start a run of just the perf tests, for dev convenience when working on them)

Reply:
I changed the test to be a standard test instead of a script, I saw that the playwright documentation always uses `npx playwright ...`, if it makes sense to change it we can do it but maybe on another pr.